RacingCarTest/Step2/CalculatorTest.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  CalculatorTest.swift
+//  RacingCarTest
+//
+//  Created by 조민호 on 2021/10/31.
+//
+
+import XCTest
+
+class CalculatorTest: XCTestCase {
+
+    let calculator = Calculator()
+    
+    func test_덧셈() throws {
+        let result = calculator.add(5, 5)
+        let expect = 10
+        
+        XCTAssertEqual(result, expect)
+    }
+
+    func test_뺄셈() throws {
+        let result = calculator.subtract(5, 5)
+        let expect = 0
+        
+        XCTAssertEqual(result, expect)
+    }
+    
+    func test_곱셈() throws {
+        let result = calculator.multiply(5, 5)
+        let expect = 25
+        
+        XCTAssertEqual(result, expect)
+    }
+    
+    func test_나눗셈() throws {
+        let result = calculator.divide(5, 5)
+        let expect = 1
+        
+        XCTAssertEqual(result, expect)
+    }
+    
+    func test_사칙연산_기호가_아닌경우() throws {
```

Comment:
사칙연산 기호가 아닐 경우에 대한 테스트를 작성하신 것은 아주 좋습니다.👍
다만 나중에 테스트 코드를 볼 때 `XCTAssert(expect == -4)` 구문에서 `-4` 가 사칙연산 기호가 아닐 때의 결과를 의미하는 지 파악하기가 힘들어 보입니다.
에러를 표현하는 방법은 다양하다고 생각합니다. 테스트 구문에서 조금 더 쉽게 이해할 수 있는 표현법이 있을까요?🤔
다양한 XCTAssert 함수들을 살펴보면 힌트를 얻을 수도 있을 것 같아요!🤗

Reply:
```suggestion
    func test_사칙연산_기호가_아닌경우() throws {
        XCTAssertThrowsError(try calculator.calculate(2, "#", 3)) { error in
            XCTAssertEqual(error as! CalculatorError, CalculatorError.valueIsNotOperator)
        }
    }
```

XCTAssertThrowsError 함수를 사용하는게 적합한 것 같아 이렇게 변경해보았습니다!